api/session/auth_session.py

```diff
@@ -91,12 +92,28 @@ def clear_session():
         session.clear()
         clear_sentry()
 
+        redirect_route = (  # TODO: Remove defaulting to Magic Links, instead use return_app
```

Comment:
On picking up this TODO, do we essentially just need to find all links to `/sessions/sign-out` & ensure they have the relevant `return_app` i.e. [application](https://github.com/communitiesuk/funding-service-design-frontend) would have `magic_links_bp.signed_out` per:

https://github.com/communitiesuk/funding-service-design-authenticator/blob/767370de417a0da19a6fb86cd474da57016ec243/frontend/magic_links/routes.py#L41-L52

Reply:
yeah it was previously just hardcoded to `magic_links_bp.signed_out` so the TODO would be to create a `SupportedApp` & `SafeAppConfig` for that configuration and then make sure any apps requesting that type of auth pass the param. We can then remove the default flow and just always require a `return_app` I imagine